src/pages/Edit/MessageEditPage.jsx

```diff
@@ -45,7 +45,7 @@ const MessageEditPage = () => {
   const handleClickChange = (e) => {
     const anonymity = name === 'anony';
     UpdateMessage(participationId, anonymity, msgText);
-    navigate(-1); //버튼 클릭시 이동할 주소
+    navigate(-1, { replace: true }); //버튼 클릭시 이동할 주소
```

Comment:
선물후기 작성 페이지도 제가 다시 생각해보니까 의미상으로 이렇게 명시하는게 좋을거같아서 이렇게 바꿔놨었는데 이렇게 하는 거 어때요?
```suggestion
    navigate(`/funding/${fundingId}`, { replace: true }); //버튼 클릭시 이동할 주소
```